pan3d/ui/axis_selection.py

```diff
@@ -0,0 +1,111 @@
+from trame.widgets import html, vuetify
+
+
+class AxisSelection(vuetify.VNavigationDrawer):
+    def __init__(self):
+        super().__init__(
+            value=("array_active",),
```

Comment:
Should __"array_active"__ be an actual parameter rather than fixed like that?

Reply:
Do you mean using `v_model`? We wouldn't want `array_active` to be set to false when the drawer is collapsed.

Reply:
What I'm saying is that the macro component should not make the choice of variable names. 

```python
class AxisSelection(vuetify.VNavigationDrawer):
    def __init__(self, drawer_visible):
        super().__init__(
            value=(drawer_visible,),
```

Reply:
But if you want to keep those fixed on purpose that is fine. You are just preventing yourself of really leveraging the flexibility of what those "macro" components means. 

So far, you are using them like static function rather than reusable piece of UI.

Reply:
Should that apply to all state variables, like the axis vars, etc?

Reply:
Changed in [5b7331b](https://github.com/Kitware/pan3d/pull/28/commits/5b7331b4dadd55dcaaa3d2650ee192bb603740af)